### PR TITLE
Use alias for astroid.nodes 03

### PR DESCRIPTION
**Pull request by cdce8p**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

## Description

Followup to #4656

### CODE REVIEW

File: pylint/checkers/utils.py [hunk 1]
```diff
@@ -770,65 +767,65 @@ def decorated_with_property(node: astroid.FunctionDef) -> bool:
...
-                    and isinstance(inferred.function, astroid.FunctionDef)
+                    and isinstance(inferred.function, nodes.FunctionDef)
                 ):
                     return decorated_with_property(inferred.function)
     return False
 
 
 def decorated_with(
-    func: Union[astroid.FunctionDef, astroid.BoundMethod, astroid.UnboundMethod],
+    func: Union[nodes.FunctionDef, astroid.BoundMethod, astroid.UnboundMethod],
```
Comment (Pierre-Sassoulas) [inline pylint/checkers/utils.py:822]: I was suspicious that astroid.bases was containing nodes too.. `BoundMethod` and `UnboundMethod` could be considered like nodes, right ?

File: pylint/checkers/utils.py [hunk 2]
```diff
@@ -1528,16 +1519,14 @@ def get_subscript_const_value(node: astroid.Subscript) -> astroid.Const:
     :raises InferredTypeError: if the subscript node cannot be inferred as a Const
     """
     inferred = safe_infer(node.slice)
-    if not isinstance(inferred, astroid.Const):
-        raise InferredTypeError(
-            "Subscript.slice cannot be inferred as an astroid.Const"
-        )
+    if not isinstance(inferred, nodes.Const):
+        raise InferredTypeError("Subscript.slice cannot be inferred as an nodes.Const")
```
Comment (Pierre-Sassoulas) [inline pylint/checkers/utils.py:1523]:
```suggestion
        raise InferredTypeError("Subscript.slice cannot be inferred as a nodes.Const")
```

File: pylint/checkers/variables.py [hunk 3]
```diff
@@ -713,9 +714,7 @@ def __init__(self, linter=None):
 
     @utils.check_messages("redefined-outer-name")
     def visit_for(self, node):
-        assigned_to = [
-            var.name for var in node.target.nodes_of_class(astroid.AssignName)
-        ]
+        assigned_to = [var.name for var in node.target.nodes_of_class(nodes.AssignName)]
```
Comment (Pierre-Sassoulas) [inline pylint/checkers/variables.py:717]:
```suggestion
        assigned_to = [a.name for a in node.target.nodes_of_class(nodes.AssignName)]
```

File: pylint/checkers/variables.py [hunk 4]
```diff
@@ -1656,10 +1644,10 @@ def _loopvar_name(self, node, name):
...
             sequences = (
-                astroid.List,
-                astroid.Tuple,
-                astroid.Dict,
-                astroid.Set,
+                nodes.List,
+                nodes.Tuple,
+                nodes.Dict,
+                nodes.Set,
                 astroid.objects.FrozenSet,
```
Comment (Pierre-Sassoulas) [inline pylint/checkers/variables.py:1651]: Look like there's other class too that should go in astroid.nodes ?

**Review by Pierre-Sassoulas — APPROVED** [review]
Great cleanup !

File: pylint/checkers/utils.py (hunk 1 above)
Comment (cdce8p) [reply pylint/checkers/utils.py:822]: `BoundMethod` and `UnboundMethod` inherit from `Proxy` not `NodeNG`. So I would say no.

File: pylint/checkers/variables.py (hunk 4 above)
Comment (cdce8p) [reply pylint/checkers/variables.py:1651]: From the `astroid.objects` docstring:
```py
"""
Inference objects are a way to represent composite AST nodes,
which are used only as inference results, so they can't be found in the
original AST tree. [...]
"""
```

As they are only used in inference, I don't think they should move to the `nodes` package. At least not for now.

**Pull request merged**